docs/tech_stack.md

```diff
@@ -51,7 +53,7 @@ When using the
 [GCP Cloud IoT Core MQTT Bridge](https://cloud.google.com/iot/docs/how-tos/mqtt-bridge#publishing_telemetry_events)
 there are multiple ways the subschema used during validation is chosen.
 * All messages have their attributes validated against the `.../attributes.json` schema. These attributes are
-automatically defined server-side by the MQTT Client ID and Topic, and are not explicitly included in any message payload.
+automatically defined server-side by the MQTT h ID and Topic, and are not explicitly included in any message payload.
```

Comment:
sneaky typo!

Reply:
h = "help, I've had to much/little coffee"